docs/kubernetes/monitoring.md

```diff
@@ -65,6 +66,16 @@ Once you have opened the UI, you can explore node stats (CPU, Memory, etc...) un
 
 ![Image of Kuberentes dashboard](../images/k8s-monitoring-dashboard.png)
 
+## Monitoring extension
+
+A quick way to scaffold out cloud-native and open source monitoring components is to use the [acs-engine monitoring extension](https://github.com/Azure/acs-engine/tree/master/extensions/prometheus-grafana-k8s). For details on how to use the monitoring extension, please refer to the extension documentation. By embedding the extension in your apimodel, the extension will do much of the work to create a monitoring solution in your cluster, which includes the following...
```

Comment:
Is it possible to make `extension documentation` a link?

Reply:
` which includes the following... ` -->`which includes the following:`

Reply:
Absolutely, changes made!